### ISSUE TITLE
Documentation appears to be incorrect.

### ISSUE THREAD

**Issue by laurenblackfox** [OmniDB/OmniDB#787]
https://omnidb.org/en/documentation-en/19-deploying-omnidb-server

Documentation states the following nginx config:

```
server {
    listen 443 ssl;
    listen [::]:443 ssl;
    include snippets/ssl-domain.conf;
    include snippets/ssl-params.conf;
    server_name domain.org;
    client_max_body_size 75M;

    location /wss {
<snip>
    }

    location / {
<snip>
    }
}
```

However, when running, a warning is generated:
`[11/29/2018 03:44:27] WARNING [django.request:152] Not Found: /ws`
Which tells me, that the websocket url generated by the app is /ws, not the stated /wss, therefore directing to the wrong endpoint.

Altering the above nginx config to use the /ws endpoint instead of /wss fixes the issue.

**Comment by rafaelthca** [comment]
hi @laurenblackfox,

Are you accessing OmniDB in the browser with HTTPS or HTTP?

If you are using HTTPS then OmniDB will connect to `wss://cloud.omnidb.org:25890/wss` and nginx needs to be configured to use `/wss`.

On the other hand, if HTTP is being used, OmniDB will connect to `ws://cloud.omnidb.org:25890/ws`, and then you should use `/ws`.

Docs indeed don't explain that, I will update it.

Regards,

Rafael.

**Comment by StephanZaat** [comment]
Hi @rafaelthca 
I also just ran into this and found it lacking in the documentation. Found the fix myself before checking here, but it might be useful to add a small note. Especially since the proxy often handles the TLS part and passes it unencrypted to the underlaying service.
Thanks!
Stephan

**Comment by felipebiox** [comment]
Hi all,

OmniDB 3 beta is now live and websocket was completely replaced with a new strategy using long polling instead, along with many other changes.

New documentation can be found here: [docs 3 beta](https://omnidb.readthedocs.io/en/latest/)
In case you still need to access the documentation for 2.x, it's available here [2.17.0](https://omnidb.readthedocs.io/en/2.17.0/)

I'm closing this ticket, but please feel free to open issues if you need.

Thanks for your contribution.

Best regards
